include/miopen/miopen.h

```diff
@@ -68,6 +68,7 @@
  * @defgroup argmax
  * @defgroup groupnorm
  * @defgroup cat
+ * @defgroup padreflection
```

Comment:
I think `PadReflection` and `PadConstant`(https://github.com/moreh-dev/MIOpen/pull/4) should be in the same group.
How about `pad`? Discuss it with @o2buzzle .

Reply:
resolved in https://github.com/moreh-dev/MIOpen/pull/7/commits/e41d851c6c8ae49ece2299504d17ca18d81ba75a